src/app/views/dashboard/stickers/stickers.ts

```diff
@@ -99,14 +101,26 @@ export default class RouteDashStickers extends BaseRouteComponent {
 	}
 
 	routeResolved($payload: InitPayload) {
-		this.balance = $payload.balance;
-		this.stickerCost = $payload.stickerCost;
+		this.fetchStickers($payload);
```

Comment:
instead of doing all of this, you can actually just call `this.reloadRoute()`. It'll essentially rerun the route resolver, recall routeResolved, and push the new payload data into the cache for the route. =]